src/classes/Carts/UserCart.ts

```diff
@@ -434,7 +434,8 @@ export default class UserCart extends Cart {
                         skuCount.mostCanTrade
                     )}`;
 
-                    this.addOurItem(sku, amount);
+                    // Add the amount we can trade and substract added amount from command (if exist)
+                    this.addOurItem(sku, this.our[sku] ? skuCount.mostCanTrade - this.our[sku] : skuCount.mostCanTrade);
```

Comment:
if line 432 `"I can only sell ${skuCount.mostCanTrade}"` is correct (which is appears to be) then I think this line needs to be

`this.addOurItem(sku, skuCount.mostCanTrade);` because that would add the most we listed that could trade.